qiskit/wrapper/_wrapper.py

```diff
@@ -364,3 +364,26 @@ def load_qasm_file(qasm_file, name=None,
         QISKitError: if the file cannot be read.
     """
     return circuit_from_qasm_file(qasm_file, name, basis_gates)
+
+
+def qobj_to_quantum_circuit(qobj):
```

Comment:
can we just call it `qobj_to_circuit` for succinctness. how i wish the class was also Circuit instead of QuantumCircuit. :)

Reply:
Done